Bottom: 0d4011886f8e5ee42bddb43ac5625bc95301161a
Top:    45118f9264c96fdae6715cdad0d19faf54ab413a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-23 00:10:58 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
